examples/config.yaml

```diff
@@ -74,10 +73,14 @@ listen: http://127.0.0.1:5555
 web_path_prefix: /
 
 # Optional kubectl version which provides a download link to the the binary
-kubectl_version: v1.11.2
+kubectl_version: v1.23.6
 
 # Optional Url to display a logo image
 # logo_uri: http://<path-to-your-logo.png>
 
 # Enable more debug
 debug: false
+
+hmac_secret: test
```

Comment:
I can drop this but dex-k8s-authenticator requires some value to be set for this